### ISSUE TITLE
Reduce Particle Use in Spells

### ISSUE THREAD

**Issue by NathanWolf** [elBukkit/MagicPlugin#582]
Since Mojang is apparently satisfied with the incredible drop in particle performance in 1.13, we will need to reduce our use of particles. Shame.

```
Mob < Serious lag
Smite
Wither on max lvl when it shoots 3 skulls < Serious lag
Arrow rain < Significant lag
Blast
Kill < Significant lag
Harden
Wave
Sandblast
Earthquake
Meteor shower
Railgun
Tornado
Singularity < Serious lag
Nuke
Dragon ball < Significant lag
```

**Comment by NathanWolf** [comment]
@GottaSlay I will have all of these reduced in the next dev build, except for Dragon Ball. The odd/interesting thing about that one is I'm not using any custom effects. What appears on the ground there are the vanilla effects for the dragon's fireball, the same as you'd get fighting the dragon in the end. So I'm not sure what to do about that one.

**Comment by GottaSlay** [comment]
Well that's great news regardless!

**Comment by NathanWolf** [comment]
Released in 7.7.5
